### PR TITLE
ego: add page

### PR DESCRIPTION
**Pull request by invakid404**
This PR adds a page for Funtoo's `ego` command-line utility.

As a Funtoo developer and user, who constantly forgets how to do certain things with ego, I thought I might as well write the most common and most useful ego commands in a tldr page :^)

### CODE REVIEW

File: pages/linux/ego.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`ego sync`
+
+- Update bootloader configuration:
+
+`ego boot update`
+
+- Read a Funtoo wiki page:
+
+`ego doc {{wiki_page}}`
```
Comment (blueskyson) [inline pages/linux/ego.md:16]: I modify this example according to https://funtoo-ego.readthedocs.io/en/develop/ref/cli/ego-doc.1.html.
```suggestion
- Read the section from Funtoo wiki page by a given string:

`ego doc {{string}}`
```

File: pages/linux/ego.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Print current profile:
+
+`ego profile show`
+
+- Enable a mix-in:
+
+`ego profile mix-in +{{mixin}}`
```
Comment (blueskyson) [inline pages/linux/ego.md:24]: Please briefly explain what is a `mix-in`. Also you can use a concrete example (such as `gnome`) to show what will be a proper argument of `mix-in`.
```suggestion
- Enable a mix-in:

`ego profile mix-in +{{gnome}}`
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Here are some suggestions.

File: pages/linux/ego.md (hunk 1 above)
Comment (invakid404) [reply pages/linux/ego.md:16]: `wiki_page` is the way it's worded in the help message for `ego doc`. It's more accurate IMHO.
I'll make it a bit more descriptive though, I agree.

File: pages/linux/ego.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# ego
+
+> Funtoo's official system personality management tool.
+> More information: <https://funtoo-ego.readthedocs.io/en/develop/index.html>.
```
Comment (CleanMachine1) [inline pages/linux/ego.md:4]:
```suggestion
> More information: <https://funtoo-ego.readthedocs.io/en/develop/>.
```

File: pages/linux/ego.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Print current profile:
+
+`ego profile show`
+
+- Enable/Disable mix-ins (various optional settings):
+
+`ego profile mix-in +{{gnome}} -{{kde-plasma-5}}`
+
+- Query Funtoo bugs, related to a given package:
```
Comment (CleanMachine1) [inline pages/linux/ego.md:26]:
```suggestion
- Query Funtoo bugs, related to a specified package:
```

File: pages/linux/ego.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Read a Funtoo wiki page by name:
+
+`ego doc {{wiki_page}}`
+
+- Print current profile:
+
+`ego profile show`
+
+- Enable/Disable mix-ins (various optional settings):
```
Comment (CleanMachine1) [inline pages/linux/ego.md:22]:
```suggestion
- Enable/Disable mix-ins:
```
Not required.

File: pages/linux/ego.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# ego
+
+> Funtoo's official system personality management tool.
+> More information: <https://funtoo-ego.readthedocs.io/en/develop/>.
+
+- Synchronize the Portage tree:
+
+`ego sync`
+
+- Update bootloader configuration:
```
Comment (CleanMachine1) [inline pages/linux/ego.md:10]:
```suggestion
- Update the bootloader configuration:
```
Maybe this will read better.

**Review by blueskyson — APPROVED** [review]
🎉

**Review by pixelcmtd — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]
lgtm

**Review by marchersimon — APPROVED** [review]
Thanks @invakid404

**Pull request merged**